protobuf/echo/test_doubles/ServiceStub.hpp

```diff
@@ -76,7 +75,7 @@ namespace services
         infra::SharedPtr<MethodDeserializer> StartMethod(uint32_t serviceId, uint32_t methodId, uint32_t size, const services::EchoErrorPolicy& errorPolicy) override;
 
     public:
-        static const uint32_t serviceId = 1;
+        static constexpr uint32_t defaultServiceId = 1;
```

Comment:
serviceId has been moved into services::Service. But do you need defaultServiceId here?